magefile.go

```diff
@@ -25,6 +25,12 @@ var cliExe = "cartridge"
 var goPackageName = "github.com/tarantool/cartridge-cli/cli"
 var packagePath = "./cli"
 
+var generateModePath = filepath.Join(packagePath, "create", "codegen", "generate_mode.go")
+var generatedFilesPath = "." + string(filepath.Separator) + filepath.Join(packagePath, "create", "codegen", "static")
```

Comment:
Please, use `filepath.Join(".", ...)` or just `fmt.Sprintf("./%s", ...)`.

Using `fileptah.Separator` is good, but look at the code above and don't care ;)